openedx/core/djangoapps/auth_exchange/tests/test_views.py

```diff
@@ -180,3 +184,47 @@ def test_dot_password_grant_supported(self):
     def test_dot_client_credentials_unsupported(self):
         access_token = self._create_dot_access_token()
         self._verify_response(access_token, expected_status_code=401)
+
+    def _create_jwt_token(self, grant_type=Application.GRANT_PASSWORD, scope='email profile'):
```

Comment:
Can you please change the following, at least in the test code.
```suggestion
    def _create_jwt_token(self, grant_type='password', scope='email profile'):
```